test/lib/jdk/test/lib/security/SecurityUtils.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
2022?

Reply:
I think 2020 is correct since the change is to restore the upstream code.